packages/labs/src/query-list/queryList.tsx

```diff
@@ -217,6 +217,12 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
         }
     }
 
+    private handlingEmptyListCorrectly() {
+        const emptyFilteredList = this.state.filteredItems.length === 0;
+        const activeItemIsUndefined = this.props.activeItem === undefined;
+        return emptyFilteredList && activeItemIsUndefined;
```

Comment:
Yeah, in the if statement condition. The fact that I have to demorgan's law on this in my head while going back and forth between two lines makes this somewhat difficult to parse.